CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+### 0.3.0 (2019-08-12)
+
+* Fix crash related to handling of settled channels (https://github.com/raiden-network/raiden-services/pull/512)
```

Comment:
can you add `Bugfix` in front of bugfixes and put them at the bottom of the list as in 0.2.0